src/java/edu/ucsd/library/dams/api/DAMSAPIServlet.java

```diff
@@ -1082,7 +1082,24 @@ else if ( path.length == 4 && path[1].equals("objects")
 					log.error("Error minting DOI", ex );
 					info = error("Error minting DOI: " + ex.getMessage(), ex);
 				}
-			}
+			} 
+            // POST /objects/bb1234567x/update_doi
```

Comment:
For editing/updating, PUT may make more senses. @mcritchlow Do we want to do that for DOI as well? Just curios.

Reply:
@lsitu - From the mint doi modified documentation, they use post.  https://support.datacite.org/reference#mint-doi

Reply:
@VivianChu Thanks for the link. I think I am fine with it.